packages/eddsa-ticket-pcd/src/EdDSATicketPCD.ts

```diff
@@ -16,6 +16,12 @@ import { getEdDSATicketData, ticketDataToBigInts } from "./utils";
 
 export const EdDSAPCDTypeName = "eddsa-ticket-pcd";
 
+export enum TicketCategory {
+  ZuConnect = 0,
+  Devconnect = 1,
+  PcdWorkingGroup = 2
+}
```

Comment:
I think we're also going to need a category for legacy Zuzalu tickets (#618) - I'll add an extra item to the enum in the PR for that.

That said, I am a bit unsure about this categorization. It seems like it repeats something we already know - the category "Devconnect" maps precisely on to "PCDs from the Devconnect feed", same with ZuConnect. I also feel like we wouldn't want the ZuConnect feed to be able to issue PCDs with the Devconnect category, but I can't see how we would prevent that.

Reply:
Another thing I would like to have for legacy Zuzalu tickets is the ability to override things like the `header` in `getDisplayOptions`, and the QR code rendering in `CardBody.tsx`. So, what I really want for legacy Zuzalu tickets is a new PCD type which is a subclass of `EdDSATicketPCD`, which differs only in the `name`, `getDisplayOptions`, and `renderCardBody` fields. 

If we can easily create subclasses of the `EdDSATicketPCD`, do we need categories at all? In cases where we would vary behaviour based on the category field, we could instead vary behaviour based on the PCD type name.

Reply:
> I think we're also going to need a category for legacy Zuzalu tickets (https://github.com/proofcarryingdata/zupass/issues/618) - I'll add an extra item to the enum in the PR for that.

Sounds good!

> That said, I am a bit unsure about this categorization. It seems like it repeats something we already know - the category "Devconnect" maps precisely on to "PCDs from the Devconnect feed", same with ZuConnect. I also feel like we wouldn't want the ZuConnect feed to be able to issue PCDs with the Devconnect category, but I can't see how we would prevent that.

I think that's the main categorization of it right now, in that Tripsha's feed will should always return `TicketCategory.Zuconnect` PCDs and our Devconnect issuance service returns `TicketCatogory.Devconnect` PCDs. How else would we be able to differentiate without an explicit field like this one? I guess we could check the eventId, but that's relatively unwieldy to do on the frontend. 

cc @ichub on the subclasses question - how difficult and useful would such a construction be?

Reply:
discussed with Ivan and subclasses are out of scope for now

Reply:
There's no mechanism for restricting the Zuconnect feed to produce only tickets with `TicketCategory.Zuconnect`. My concern is that we now have two ways of grouping tickets: by feed, and by category (there's even a third: by folder). Since we already have feeds, and categories are extra work, do we really need them?